src/components/Modal/Modal.styles.js

```diff
@@ -1,11 +1,63 @@
 import styled from 'styled-components';
 
 export const StyledModal = styled.div`
-	background: #205081;
+	background: ${props => {
+		if (props.type === 'jira') {
+			return `#205081`;
+		} else if (props.type === 'bitbucket') {
+			return `#205081`;
+		} else if (props.type === 'forecast') {
+			return `#f36c00`;
+		}
+	}}
 	border-radius: 10px;
+	color: var(--color-white);
 	left: 50%;
 	padding: var(--spacing-large);
 	position: fixed;
 	top: 50%;
 	transform: translate(-50%, -50%);
+	z-index: 3;
+	input {
```

Comment:
These nested selectors should be split into styled components

Reply:
All nested selectors from this styles.js file are now styled components in the SettingsPanel.styles.js